src/js/deprecations/deprecate-coblocks-media-card.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { getBlockType, registerBlockType, unregisterBlockType } from '@wordpress/blocks';
+import domReady from '@wordpress/dom-ready';
+
+/**
+ * Prevents the CoBlocks Media Card block from being insertable.
+ */
+function deprecateCoBlocksMediaCardSettings() {
+	const coreMediaText = getBlockType( 'core/media-text' );
+	const coBlocksMediaCard = getBlockType( 'coblocks/media-card' );
+	if ( mediaCardBlockDeprecated( coreMediaText, coBlocksMediaCard ) ) {
+		unregisterBlockType( 'coblocks/media-card' );
+		registerBlockType( 'coblocks/media-card', {
+			...coBlocksMediaCard,
+			supports: {
+				...coBlocksMediaCard.supports,
+				inserter: false,
+			},
+		} );
+	}
+}
+
+/**
+ * Check whether or not core/media-card is available for use.
+ *
+ * @param {Object} coreMediaText The results of getBlockType.
+ * @param {Object} coBlocksMediaCard The results of getBlockType.
+ *
+ * @return {boolean} True or false if block is deprecated.
+ */
+export function mediaCardBlockDeprecated( coreMediaText, coBlocksMediaCard ) {
+	if ( ! coreMediaText ) {
+		return false;
+	}
+
+	if ( ! coBlocksMediaCard ) {
+		return false;
+	}
+	return true;
```

Comment:
@AnthonyLedesma We can probably combine the two checks here.

```js
if ( ! coreMediaText || ! coBlocksMediaCard ) {

   return false;

}
```

or maybe even:

```js
return ( ! coreMediaText || ! coBlocksMediaCard ) ? false : true;

Reply:
Good call. Keep it simple 👍